sycl/include/sycl/reduction.hpp

```diff
@@ -2386,8 +2347,8 @@ void reduction_parallel_for(handler &CGH,
       device D = detail::getDeviceFromHandler(CGH);
 
       if (D.has(aspect::atomic64)) {
-        reduCGFuncAtomic64<KernelName>(CGH, KernelFunc, Range, Properties,
-                                       Redu);
+        reduCGFuncForNDRangeBothFastReduceAndAtomics<KernelName>(
```

Comment:
If there's no longer a difference between the two, would it make sense to keep the simpler name, i.e. `reduCGFuncAtomic64`?

Reply:
I don't think so. First, the longer name matches used features better. Second, I anticipated relatively huge changes with naming of these individual implementations (not limited to this particular one) in one of the next PRs.